ext/ffi_c/Function.c

```diff
@@ -172,6 +154,33 @@ function_free(Function *fn)
     xfree(fn);
 }
 
+static const rb_data_type_t function_data_type = {
+    .wrap_struct_name = "FFI::Function",
+    .function = {
+        .dmark = function_mark,
+        .dfree = function_free,
+        .dsize = NULL,
```

Comment:
Unfortunately no, that's why I left it, there are some associated structures that may or may not be set etc.

But yes I agree that it would be good to implement the dsize functions for all the types.